frontend/src/components/pages/details/Details.tsx

```diff
@@ -569,6 +544,30 @@ export const DetailsUIWithoutContext: React.FC<Props> = ({ slug, parentId, langu
                       );
                     }
 
+                    // Custom HTML templates
+                    if (section.template) {
+                      return (
+                        <section
+                          key={section.name}
+                          ref={sectionRef[section.name]}
+                          id={`details_${section.name}_ref`}
+                        >
+                          <DetailsSection
+                            htmlId={`details_${section.name}`}
+                            titleId={`details.${section.name}`}
+                            className={marginDetailsChild}
+                          >
+                            <HtmlParser
+                              template={section.template}
+                              id={details.id.toString()}
+                              type="trek"
+                              cityCode={details.cities_raw[0]}
```

Comment:
On the deleted lines, there's a details.cities_raw?.[0] but not here, cities_raw being undefined is not possible anymore?

Reply:
Great point!
`cities_raw` can't be undefined but the array can be empty and i don't handle this case.

If the custom template is city-dependent and this one returns `undefined`, it's probably necessary to not display this section at all (e.g. a forecast widget).

I'll have to think about it before going any further.

Reply:
The [last commit](https://github.com/GeotrekCE/Geotrek-rando-v3/pull/988/commits/e52fb05962643297490efa682bf44dd27b3a2469) allows to check is all used variables are defined to display the custom section.